gr-soapy/grc/soapy_sdrplay_source.block.yml

```diff
@@ -55,12 +55,26 @@ parameters:
     default: 'False'
     hide: part
 
+  - id: agc_setpoint
+    label: AGC Setpoint'
+    category: RF Options
+    dtype: int
+    default: '-30'
+    hide: part
+
   - id: gain
-    label: 'RF Gain (-47dB - 0dB)'
+    label: 'Gain (0dB - 39dB)'
     category: RF Options
-    dtype: real
-    default: '-20'
-    hide: ${'all' if agc else 'part'}
+    dtype: int
+    default: '20'
+    hide: part
```

Comment:
Could hide manual gain control like before when agc is enabled, to be more clear they're mutually exclusive?

Reply:
That made it difficult to build a flowgraph with GUI controls for agc and gain, since either setpoint or gain would be hidden based on mode.

Reply:
When using GUI elements (radio buttons to select agc) hiding the fields makes it difficult to set up, so I'll keep them all visible.